explore-analyze/visualize/alert-panels.md

```diff
@@ -0,0 +1,42 @@
+---
+applies_to:
+  stack: ga
+  serverless: ga
+products:
+  - id: kibana
+---
+
+# Alert panels
+
+To view alerts in a dashboard, add **Alerts** panels that show selected alerts. Each panel can display either **Observability** or **Security** alerts with the rule tags and rule types that you select. 
+
+## Create an alerts panel
+
+1. From your dashboard, select **Add panel**.
+2. In the **Add panel** flyout, select **Alerts**. The configuration flyout appears.
+3. Under **Solution**, select either **Observability** or **Security** to specify the type of alerts you want to display.
```

Comment:
This step doesn't apply in serverless

Reply:
```suggestion
3. Under **Solution**, select either **Observability** or **Security** to specify the type of alerts you want to display. In serverless projects, you don't have to select a solution because you're already in a solution context.
```
Something like that for example. Feel free to rephrase my sentence isn't great :D